auth0/internal/debug/debug.go

```diff
@@ -1,36 +0,0 @@
-package debug
```

Comment:
Never used anywhere, so we're cleaning it up so we don't need to upgrade this to sdk v2 as well.